doc/qbittorrent.1

```diff
@@ -1,41 +1,435 @@
+.TP
+\f[B]\f[CB]--configuration=\f[B]\f[R]\f[I]NAME\f[R]
+Use a different configuration with name \f[I]NAME\f[R].
+This enables running multiple separate instances of qBittorrent at the
+same time without conflict (provided they don\[cq]t listen on the same
+ports for WebUI access, incoming connections, etc; in this case, the
+additional instances will either fail to start or be functionally
+unusable).
+.RS
+.PP
+Configuration and data files will be stored under a configuration
+hierarchy similar to the default one (see the \f[I]FILES\f[R] section
+for more information) but with a final directory named
+\f[B]qBittorrent_\f[BI]NAME\f[B]\f[R] instead of \f[B]qBittorrent\f[R].
+.PP
+For example, while the main configuration file is stored as
+\f[I]\[ti]/.config/qBittorrent/qBittorrent.conf\f[R], the configuration
+file for a configuration named \f[B]foo\f[R] will be stored as
+\f[I]\[ti]/.config/qBittorrent_\f[BI]foo\f[I]/qBittorrent.conf\f[R].
+.PP
+If a configuration with the given \f[I]NAME\f[R] does not exist yet, it
+is created with default settings.
+If it already exists, it is loaded as a normal configuration would be.
+.PP
+If both \f[B]\f[CB]--configuration\f[B]\f[R] and
+\f[B]\f[CB]--profile\f[B]\f[R] are specified, only
+\f[B]\f[CB]--profile\f[B]\f[R] takes effect.
+.RE
+.TP
+\f[B]\f[CB]--no-splash\f[B]\f[R]
+Do not display the splash screen on startup.
+.TP
+\f[B]\f[CB]--profile=\f[B]\f[R]\f[I]PATH\f[R]
+Store configuration \f[I]FILES\f[R] in \f[I]PATH\f[R].
+Similarly to \f[B]\f[CB]--configuration\f[B]\f[R], this option can be
+used to run multiple separate instances of qBittorrent at the same time
+without conflitcts (the same caveats about listening port conflicts and
+other potential unavoidable conflicts apply).
+.RS
+.PP
+The difference is that configuration files are all stored together under
+the specified configuration \f[I]PATH\f[R], instead of scattered under a
+separate directory hierarchy in the system\[cq]s configuration
+directories (see the \f[I]FILES\f[R] section for more information).
+.PP
+If a special directory literally named \f[B]\f[CB]profile\f[B]\f[R]
+exists in the \f[I]same directory as the qBittorrent executable\f[R],
+executing qBittorrent with no \f[B]\f[CB]--profile=\f[B]\f[R] option is
+the same as executing qBittorrent from the qBittorrent executable
+directory with the options
+\f[B]\f[CB]--profile=profile --relative-fastresume\f[B]\f[R].
+This special case is known as the \[lq]portable mode\[rq] in
+qBittorrent.
+Note that if this special directory is present in the qBittorrent
+executable directory, but a non-empty \f[B]\f[CB]--profile=\f[B]\f[R] is
+specified, the command line option will take precedence and
+\f[B]\f[CB]--relative-fastresume\f[B]\f[R] will not be implied.
+Specifying a non-empty \f[B]\f[CB]--profile=\f[B]\f[R] together with
+\f[B]\f[CB]--relative-fastresume\f[B]\f[R] explicitly is functionally
+equivalent to the \[lq]portable mode\[rq], but under a profile diretory
```

Comment:
"directory" - typo

Reply:
@glassez fixed. PS - you can review the markdown document, it's more readable. the manpage itself is autogenerated with pandoc.